LICENSE-APACHE

```diff
@@ -180,7 +180,7 @@ APPENDIX: How to apply the Apache License to your work.
    To apply the Apache License to your work, attach the following
    boilerplate notice, with the fields enclosed by brackets "[]"
    replaced with your own identifying information. (Don't include
-   the brackets!)  The text should be enclosed in the appropriate
+   the brackets!) The text should be enclosed in the appropriate
```

Comment:
I hope this is not a problem as this copy of https://www.apache.org/licenses/LICENSE-2.0.txt differs in many-many whitespaces.